php/README.md

```diff
@@ -10,6 +10,7 @@ Our main focus will be the use of [Symfony2](http://symfony.com/) framework.
 1. [DocBlocks](#docblocks)
 1. [Visibility](#visibility)
 1. [Namespace and Use declarations](#namespace-and-use-declarations)
+1. [Control Structures](#control-structures)
```

Comment:
The order is intentional, since GitHub will automatically create the numerical list.